tests/functional/common.js

```diff
@@ -54,31 +55,23 @@ export function withTempAddonDir(
 }
 
 
-// reportRunCommandError helper
+// reportCommandErrors helper
 
-export function reportRunCommandError(obj: Object, msg: ?string) {
+export function reportCommandErrors(obj: Object, msg: ?string) {
   const errorMessage = msg || 'Unexpected web-ext functional test result';
-  const formattedErrorData = JSON.stringify(obj, null, 2);
+  const formattedErrorData = prettyjson.render(obj).replace('\\n', '\n');
```

Comment:
This seems to log the object twice. It logs it once and then logs it again as an error nested within the first object's `stderr`. Can you try to track down why that's happening? It should only log the error once.